aries-site/src/examples/components/controls/TabsExample.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Tab, Tabs } from 'aries-core';
+import { Box } from 'grommet';
+
+export const TabsExample = () => {
+  const [index, setIndex] = React.useState();
+  const onActive = nextIndex => setIndex(nextIndex);
+
+  return (
+    <Box pad="large" background="background-front">
```

Comment:
Noticing that the design mock does not have this padding...  For consistency, should we consider using `<UsageExample>` for each of these examples which would provide consistent padding from example to example, plus support light and dark themeMode illustrations ([similar](https://github.com/hpe-design/aries/pull/185/files#r365909462)) ?

Reply:
We can remove this from Tabs example to match the design for now. I noticed the switch in the Accordion example to have the background with padding and I thought maybe the Tabs would use that as well. For now, I will change it to be consistent with the designs. If a background/padding is added in the future I agree that UsageExample should be used.